lightning/src/routing/router.rs

```diff
@@ -728,7 +729,7 @@ pub fn get_route<L: Deref>(our_node_id: &PublicKey, network: &NetworkGraph, paye
 							}
 
 							let new_graph_node = RouteGraphNode {
-								pubkey: $src_node_id,
+								pubkey: PublicKey::from_slice(&$src_node_id).map_err(|_| LightningError{err: format!("Public key {:?} is invalid", &$src_node_id), action: ErrorAction::IgnoreAndLog(Level::Trace)})?,
```

Comment:
I think `RouteGraphNode::pubkey` needs to be a NodeId as well.

Reply:
Yes, of course! 🤦‍♂️ Thanks! 😃